client/ayon_flame/plugins/create/create_workfile.py

```diff
@@ -14,22 +18,66 @@ class CreateWorkfile(AutoCreator):
     identifier = "io.ayon.creators.flame.workfile"
     label = "Workfile"
     product_type = "workfile"
-
+    icon = "fa5.file"
     default_variant = "Main"
 
-    def collect_instances(self):
+    @staticmethod
+    def _get_project_workfile_filepath():
```

Comment:
Shouldn't this be function somehwere in `ayon_flame.api`? (I have no context so maybe not)

Reply:
This hopefully will remain very specific to `create_workfile` so I think it's better here.

To provide more context, ideally we would have used the Flame WireTap SDK to handle this, but because of this issue: https://github.com/ynput/ayon-flame/pull/34 , we had to go with a side-car json file implementation instead.